aries-site/src/data/structures/tokens.js

```diff
@@ -38,6 +39,24 @@ export const tokens = [
     ],
     tags: [],
   },
+  {
+    name: 'Component states',
+    type: 'Getting started',
+    description:
+      'The state of a component communicates feedback based on user interaction, system status, or both.',
+    preview: {
+      component: () => <ButtonStates size="small" tabIndex={-1} />,
```

Comment:
Was expecting to see focus applied to the focus state button.

Reply:
<img width="495" alt="Screenshot 2025-02-10 at 3 49 04 PM" src="https://github.com/user-attachments/assets/64d25fb3-d7e0-44f7-bab1-a2331df4d33e" />

Same comment as below: https://github.com/grommet/hpe-design-system/pull/4632#discussion_r1950053245

Reply:
yarn build design-tokens resolved.